### PR TITLE
checking if we should scroll in console panel by whether or not node is in view now, not by node type

### PR DESCRIPTION
**Pull request by karina-y**

### CODE REVIEW

**Comment by karina-y** [comment]
tests:
stepping-06 times out in ci and locally
console_warp-01 times out in ci and passes locally
logpoint-04 times out in ci and locally

**Comment by karina-y** [comment]
is the eval being appended to the top of the console?

**Comment by karina-y** [comment]
@jasonLaster just tested this branch and evals aren't being appended to the top. i also went back to the loom and if you look closely you can see the scrollbar move as well, i was just already close to the top on that recording.

**Comment by karina-y** [comment]
latest test update:
inspector-03 times out in ci but passes locally on master and eval-scroll

File: src/devtools/client/webconsole/components/Output/ConsoleOutput.js [hunk 1]
```diff
@@ -101,6 +101,15 @@ class ConsoleOutput extends Component {
       return;
     }
 
+    //check if node is already in view
```
Comment (jaril) [inline src/devtools/client/webconsole/components/Output/ConsoleOutput.js:104]: _Very_ minor nit - I'd be more descriptive here with something like:
```js
// Don't scroll to the evaluation result if it's already in view.
```

**Review by jaril — APPROVED** [review]
Just had one small nit but apart from that this looks good! There's one failing test but it appears to pass locally. Let's ship it!

**Pull request merged**